components/service/glean/README.md

```diff
@@ -80,6 +80,10 @@ class SampleApplication : Application() {
 Once initialized, if collection is enabled, glean will automatically start collecting [baseline metrics](metrics.yaml)
 and sending its [pings](docs/pings.md).
 
+Glean should be initialized as soon as possible, and importantly, before any
+other libraries in the application start using Glean. Library code should never
```

Comment:
We setup logging and crash reporting before glean :P

Reply:
I think logging would be fine.  If crash reporting ever started using glean, we might need to change that.  Would that be hard to do?

Reply:
It's probably never going to call Glean from its own initializing code. But let's see once we get there. :)